test/e2e/apimachinery/resource_quota.go

```diff
@@ -1524,3 +1609,33 @@ func waitForResourceQuota(c clientset.Interface, ns, quotaName string, used v1.R
 		return true, nil
 	})
 }
+
+// updateResourceQuotaUntilUsageAppears updates the resource quota object until the usage is populated
+// for the specific resource name.
+func updateResourceQuotaUntilUsageAppears(c clientset.Interface, ns, quotaName string, resourceName v1.ResourceName) error {
+	return wait.Poll(framework.Poll, 1*time.Minute, func() (bool, error) {
+		resourceQuota, err := c.CoreV1().ResourceQuotas(ns).Get(quotaName, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		// used may not yet be calculated
+		if resourceQuota.Status.Used == nil {
+			return false, nil
+		}
+		// verify that the quota shows the expected used resource values
+		_, ok := resourceQuota.Status.Used[resourceName]
+		if ok {
+			return true, nil
+		}
+
+		current := resourceQuota.Spec.Hard[resourceName]
+		current.Add(resource.MustParse("1"))
```

Comment:
this need to set something that varies each time... setting the same value will no-op in etcd and result in no watch events being delivered

Reply:
Doesn't it add `1` every time?

Reply:
ah, yes. I missed the `Add`... I thought it was setting to "1" every time